controllers/pravega_controller.go

```diff
@@ -211,6 +211,9 @@ func makeControllerPodSpec(p *api.PravegaCluster) *corev1.PodSpec {
 	if p.Spec.Pravega.ControllerInitContainers != nil {
 		podSpec.InitContainers = append(podSpec.InitContainers, p.Spec.Pravega.ControllerInitContainers...)
 	}
+	if p.Spec.Pravega.ControllerEnvVars != nil {
+		podSpec.Containers[0].Env = p.Spec.Pravega.ControllerEnvVars
+	}
```

Comment:
Should not we rather append it to the existing array, like it's done for segment store?
https://github.com/pravega/pravega-operator/blob/74333affef845b63c2c1df5c6e138e5888427f45/controllers/pravega_segmentstore.go#L263

Reply:
For controller pods, there is no `Env` added to podspec as in segmentstore. That is the reason added like this

Reply:
Thanks @jkhalack for pointing this. I have modified it as suggested